src/fuzz_introspector/datatypes/function_profile.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, elem: Dict[Any, Any]) -> None:
         self.new_unreached_complexity: int = 0
         self.total_cyclomatic_complexity: int = 0
 
+        # Saving callsitecalltree for unreachable function
+        self.callsite = self.load_unreachable_function_callsite(elem['Callsites'])
```

Comment:
`load_unreachable_function_callsite` seems not appropriately named? The function loads all callsites into a list, what does the `unreachable` refer to?

Reply:
changed

Reply:
> The name is the same?

this was an outdated comment. it's resolved.